Unix/protocol/protocol.c

```diff
@@ -1310,18 +1314,18 @@ static MI_Boolean _ProcessCreateAgentMsg(
     return MI_FALSE;
 }
 
-/* Creates and sends PamCheckUserMsg request message */
-static MI_Boolean _SendPamRequest(
-    ProtocolSocket *h,
+/* Creates and sends PamCheckUserReq request message */
+static MI_Boolean _SendPamCheckUserReq(
     const char *user,
     const char *passwd,
-    const char *file
+    MI_Uint64 handle
     )
 {
-    PamCheckUserMsg *req = NULL;
+    PamCheckUserReq *req = NULL;
     MI_Boolean retVal = MI_TRUE;
+    ProtocolSocket *h = s_permanentSocket;
```

Comment:
Can you rename this stack variable please! 'h' confused me greatly. Using the global variable directly would have been more obvious